solarforecastarbiter/metrics/preprocessing.py

```diff
@@ -89,10 +89,14 @@ def resample_and_align(fx_obs, data, tz):
                                        label=closed,
                                        closed=closed).mean()
 
+    # Remove NaNs
+    obs_resampled = obs_resampled.dropna(how="any")
+    data[fx] = data[fx].dropna(how="any")
```

Comment:
probably a bad idea to modify the data dictionary. Instead add the ``.dropna`` before align below or make some temporary variable.

Reply:
Revised as suggested.